### ISSUE TITLE
Add Post#hasContent

### ISSUE THREAD

**Issue by rlivsey** [bustlelabs/mobiledoc-kit#335]
Currently Post#isBlank is false if it has a section, even if there's no content in that section.

I'm currently doing this to check if a post has blank content:

```javascript
 function editorPostIsBlank(editor) {
  const post = editor.post;
  if (post.isBlank) {
    return true;
  }

  const sections = post.sections;
  if (sections.isEmpty) {
    return true;
  }

  if (sections.length === 1 && sections.head.isBlank) {
    return true;
  }

  return false;
}
```

**Comment by bantic** [comment]
This makes sense to me. I think it may also make sense to change `post#isBlank` to work the way `editorPostIsBlank` above does, too. Sections currently are "blank" if they have no sections OR if all their sections are blank, so it is inconsistent that posts define "blank"-ness differently.

We currently use `post#isBlank` to deal with placeholder text ([inserting an empty markup section under the cursor when typing in a blank post](https://github.com/bustlelabs/mobiledoc-kit/blob/master/src/js/editor/editor.js#L629-L631)) and a few other niche cases. Those can likely be fixed up to work with a more consistent version of `post#isBlank` instead.

Thoughts @mixonic ?

**Comment by mixonic** [comment]
I think we could introduce `isEmpty` for the case of a post having no sections, and `isBlank` for the all-contents-are-blank case. Of course I don't think we need the latter internally? It would just be public API.

Do you want to public API @rlivsey? Seems ok to have both to me.

**Comment by bantic** [comment]
released in [v0.10.7](https://github.com/bustlelabs/mobiledoc-kit/releases/tag/v0.10.7)
